tests/enforcer/gce_firewall_enforcer_test.py

```diff
@@ -458,6 +529,11 @@ def test_valid(self):
         self.assertTrue(
             self.firewall_rules._check_rule_before_adding(self.test_rule))
 
+    def test_valid_callback_false(self):
+        """Verify valid rules returns True."""
+        self.firewall_rules._add_rule_callback = lambda _: False
+        self.assertFalse(self.firewall_rules._check_rule_before_adding(self.test_rule))
```

Comment:
Nit: ensure line is no longer than 80 chars. Here and any line below that is too long as well.

Reply:
Done